src/index.js

```diff
@@ -0,0 +1,219 @@
+}
+
+const addTemp = () => { //adds 1 to temp
+    state.tempCount += 1;
+    changeTemp()
+};
+
+const subtractTemp = () => { //subtracts 1 from temp
+    state.tempCount -= 1;
+    changeTemp()
+};
+
+//this function sets ranges for temp and changes color
+//of temp number and prints emojis to landscape
+const tempRange = () => {
+    if (state.tempCount <= 49){
+        const temp = document.getElementById("temp-display")
+        temp.className ="teal";
+        const landscape = document.getElementById("landscape-container")
+        landscape.textContent = landscapeImg["5"];
+    }
+    else if (state.tempCount <= 59 && state.tempCount >= 50){
+        const temp = document.getElementById("temp-display")
+        temp.className ="green";
+        const landscape = document.getElementById("landscape-container")
+        landscape.textContent = landscapeImg["4"];
+    }
+    else if (state.tempCount <= 69 && state.tempCount >=60){
+        const temp = document.getElementById("temp-display")
+        temp.className ="yellow";
+        const landscape = document.getElementById("landscape-container")
+        landscape.textContent = landscapeImg["3"];
+    } 
+    else if (state.tempCount  >= 70 && state.tempCount <= 79){
+        const temp = document.getElementById("temp-display")
+        temp.className ="orange";
+        const landscape = document.getElementById("landscape-container")
+        landscape.textContent = landscapeImg["2"];
+    }
+    else if (state.tempCount >= 80){
+        const temp = document.getElementById("temp-display")
+        temp.className ="red";
+        const landscape = document.getElementById("landscape-container")
+        landscape.textContent = landscapeImg["1"];
+    }
+}
+
+
+
+//display state.tempCount in temp container
+const displayTemp = () => {
+    const tempContainer = document.querySelector("#temp-display")
+    tempContainer.textContent = `${state.tempCount}`;
+}
+
+//registering all event handlers
+const registerEventHandlers = () => {
+
+    const addTempButton = document.getElementById("add-temp").addEventListener("click", addTemp);
+    
+    const lowerTempButton = document.getElementById("sub-temp").addEventListener("click",subtractTemp );
+
+    updateCityHeader();
+    const updateCity = document.getElementById("cityInput");
+    updateCity.addEventListener('input', updateCityHeader)
+
+    const skySelector = document.getElementById("sky-menu");
+    skySelector.addEventListener("change",updateSky);
+
+    const resetNameButton = document.getElementById("reset");
+    resetNameButton.addEventListener("click", resetCityName)
+
+    const updateCurrentTemp = document.getElementById("real-time-temp");
+    updateCurrentTemp.addEventListener("click", updatedCurrentTemp);
+};
+
+document.addEventListener("DOMContentLoaded", registerEventHandlers);
+
+
+//change temperature from K to F
+const convertTemp = (temp) => {
+    return Math.floor((temp - 273.15) * (9 / 5) + 32);
+  };
```

Comment:
Nice! Small nit, would move this helper function above our eventListener code because they should go last!